README.md

```diff
@@ -62,3 +62,28 @@ listtodo = list id for the "Todo" list
 listdoing = listid for the "Doing" list
 listdone = listid for the "Done" List
 
+#Testing
```

Comment:
In markdown (.md files), you need a space between the hash and the word for it to be a header.
You can see the formatting in VSCode via the "preview" button in the top right, or on GitHub https://github.com/Cliver67/DevOps-Course-Starter/blob/0c061e193a1b9d0d22caf2e9db31bebd1e6feaad/README.md

Reply:
cleaned up the formatting , some readability improvements